carbonmark/components/Stats/index.tsx

```diff
@@ -13,6 +13,7 @@ interface Props {
   currentSupply?: Project["currentSupply"];
   totalRetired?: Project["totalRetired"];
   description: string;
+  projectAddress?: string;
```

Comment:
Just wondering why this is optional?
From the API it can not be, I guess.
Also constructing a link to polygonScan would not work if this undefined, right? 🤔

Reply:
Stats is also used in the portfolio sidebar that didn't require the projectAddress; if it wasn't optional it was breaking the build